.gitmodules

```diff
@@ -1,7 +1,7 @@
 [submodule "lib/J3DUltra"]
 	path = lib/J3DUltra
-	url = https://github.com/JoshuaMKW/J3DUltra.git
-    branch = aabb-support
+	url = https://github.com/HazardousPeach/J3DUltra.git
```

Comment:
As a more permanent solution, we should look into getting whatever Linux ports you may have applied here into a PR on the main repository hosted by user SageOfMirrors. The goal would be that we don't have to reference forks so updates become easier.

Reply:
Yeah that makes sense. I guess, why don't I pull request my small fixes into your branch, so we can get those merged together regardless of how responsive SageOfMirrors is, and then you pull request from your repo on to upstream? I could pull request mine directly to upstream too, but not sure how long it'll take to get merged (and I can't account for your changes).

Reply:
Let's go with merging into my fork for now as an intermediary step towards proper upstream.

Reply:
Okay, you should be able to merge that by just accepting this pull request: https://github.com/JoshuaMKW/J3DUltra/pull/1 . Then, I can set the URL back to your fork and we can merge this one?

Reply:
I've merged the PR 👍